learning/modules/quantum-mechanics/getting-started-with-qiskit.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4046fd82",
+   "metadata": {},
+   "source": [
+    "{/* cspell:ignore checkmark ccnot */}\n",
+    "\n",
+    "# Getting started with Qiskit in the classroom"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a84d2a96-7885-4f6c-b34a-f2d386d4e120",
+   "metadata": {},
+   "source": [
+    "For this Qiskit in Classrooms module, students must have a working Python environment with the following packages installed:\n",
+    "- `qiskit` v2.1.0 or newer\n",
+    "- `qiskit-ibm-runtime` v0.40.1 or newer\n",
+    "- `qiskit-aer` v0.17.0 or newer\n",
+    "- `qiskit.visualization`\n",
+    "- `numpy`\n",
+    "- `pylatexenc`\n",
+    "\n",
+    "To set up and install the packages above, see the [Install Qiskit](/docs/guides/install-qiskit) guide.\n",
+    "In order to run jobs on real quantum computers, students will need to set up an account with IBM Quantum&reg; by following the steps in the [Set up your IBM Cloud account](/docs/guides/cloud-setup) guide.\n",
+    "\n",
+    "This module was tested and used 2 seconds of QPU time. This is an estimate only. Your actual usage may vary."
```

Comment:
Can you add which processor it took 2 seconds on? e.g.,

"This module was tested and used 2 seconds of QPU time on a Heron v2 processor."